features/step_definitions/start_new_group_steps.rb

```diff
@@ -86,7 +86,7 @@
   page.should have_content 'can\'t be blank'
 end
 
-Then(/^the example content should be created$/) do
+Then(/^the example content should be present$/) do
```

Comment:
Why change created -> present? Should it be consistent with 'the example content should not be created'?